Dockerfile

```diff
@@ -4,13 +4,17 @@ RUN apt-get update && \
     apt-get install -y cmake libclang-dev && \
     rm -rf /var/lib/apt/lists/*
 
-ARG SUI_RELEASE
-
 WORKDIR /usr/src/sui
-RUN curl -sL https://github.com/MystenLabs/sui/archive/refs/tags/${SUI_RELEASE}.tar.gz | \
-    tar -xzv --strip-components 1
 
-ENV GIT_REVISION=${SUI_RELEASE}
+# ARG SUI_RELEASE
+# RUN curl -sL https://github.com/MystenLabs/sui/archive/refs/tags/${SUI_RELEASE}.tar.gz | \
+#     tar -xzv --strip-components 1
+# ENV GIT_REVISION=${SUI_RELEASE}
```

Comment:
nit: can remove the comments if not used or serving a doc purpose.

I also noticed previously `GIT_REVISION` was set as an ENV, but not used anywhere. Was this needed for some reason? And do we need the same persistence of the env var for `SUI_GIT_REVISION`?

Reply:
I kept it here for now to document the alternative approach to build from release source tarball (which is what we've been doing thus far) in case we want to fallback to it for some reason. The `GIT_REVISION` env is required by the build process, which can be detected automatically in a git repo. We had to set that manually in the previous approach.